optaweb-vehicle-routing-frontend/src/components/TspMap.tsx

```diff
@@ -60,7 +60,7 @@ const TspMap: React.SFC<ITspMapProps> = ({
       center={center}
       zoom={zoom}
       onClick={clickHandler}
-      style={{ width: '100vw', height: '100vh' }}
+      style={{ width: '100%', height: '100vmin' }}
```

Comment:
Let's use 100% height as well. It gives the expected result at least on my machine. The map fits nicely in to the available space in both directions, no scroll bars.

Reply:
Hmm, there are some issues with `height: 100%`. `style={{ width: '100%', height: 'calc(100vh - 216px)' }}` works best for me, although there is the `calc()` function again :slightly_frowning_face:. I think we need to find a way how to create a container that takes all available vertical space (window height minus navigation bar).